tests/views/layout.jade

```diff
@@ -15,6 +15,6 @@ html
         script(src='../bower/jasmine/lib/jasmine-core/jasmine-html.js')
         script(src='../bower/jasmine/lib/jasmine-core/boot.js')
         script(src='../bower/jasmine-jsreporter/jasmine-jsreporter.js')
-        script(src='../dist/db.min.js')
+        script(src='../dist/db.js')
```

Comment:
You should be getting source maps information so debugging won't be an issue. Are the source maps not working that you need to do this?